.gitignore

```diff
@@ -49,3 +49,4 @@ src/utils/quicklinks.json
 
 # IDE
 .idea
+src/pages/list.json
```

Comment:
Move it to `src/utils/quicklinks.json` for consistence

Reply:
how?